platform/Span.h

```diff
@@ -296,9 +296,9 @@ struct Span {
      *
      * @param other The Span object used to construct this.
      *
-     * @note For span with a positive extent, this function is not accessible.
+     * @note For Span with a positive extent, this function is not accessible.
      *
-     * @note OtherElementType(*)[] should be convertible to ElementType(*)[].
+     * @note OtherElementType(*)[] is convertible to ElementType(*)[].
```

Comment:
That's the enunciation of a requirement; not the enunciation of the fact. What form should be used ?

Reply:
Oh, I see. What do you think of "must be"?

Reply:
That is more precise 👍 .